Dockerfile.render-demo

```diff
@@ -13,17 +17,15 @@ RUN pnpm install --frozen-lockfile
 COPY ./frontend/ .
 
 # Build frontend
-RUN pnpm release-docker
+RUN pnpm "${RELEASE}-docker"
 
 FROM golang:1.18.4 as backend
 
-ARG VERSION="development"
-ARG GO_VERSION="1.18.4"
-ARG RENDER_GIT_COMMIT="unknown"
+ARG GO_VERSION="unknown"
+ARG GIT_COMMIT="unknown"
```

Comment:
RENDER_GIT_COMMIT is a render specific env variable https://render.com/docs/environment-variables

WIll this cause issue?

Reply:
SG. I'll revert that.